tools/src/main/java/org/apache/kafka/tools/MetadataQuorumCommand.java

```diff
@@ -131,22 +140,27 @@ private static void addDescribeParser(Subparsers subparsers) {
             .addArgument("--status")
             .help("A short summary of the quorum status and the other provides detailed information about the status of replication.")
             .action(Arguments.storeTrue());
+
         ArgumentGroup replicationArgs = describeParser.addArgumentGroup("Replication");
         replicationArgs
             .addArgument("--replication")
             .help("Detailed information about the status of replication")
             .action(Arguments.storeTrue());
+        replicationArgs
+            .addArgument("-hr", "--human-readable")
+            .help("Print human-readable timestamps")
```

Comment:
Since we're not printing timestamp with `-hr` option, maybe change to:
nit: Print human-readable timestamps duration.

Reply:
What about "Human-readable output"? It's short, clear and allows reuse. In the future and even in other tools, we could use this flag to make some fields readable or filter out less important details that make the output hard to read.